snap/snapcraft.yaml

```
@@ -5,8 +5,78 @@ summary: Snap for NVIDIA DCGM and DCGM exporter
 license: Apache-2.0
 contact: solutions-engineering@lists.canonical.com
 description: |
-  This snap includes NVIDIA DCGM and DCGM exporter to provide GPU monitoring via
-  Prometheus metrics. This snap does not include the dcgmproftester.
+  This snap includes NVIDIA DCGM and DCGM-exporter to monitor GPU via Prometheus metrics
+  A use can setup a Grafana dashboard to visualize the metrics.
+
+  The snap includes the following components:
+  - DCGM: Data Center GPU Manager
+  - DCGM-exporter: A Prometheus exporter for DCGM metrics
+  - NV-Hostengine: A service that communicates with the NVIDIA GPU driver to get the metrics
+
+  **Note**: The `DCGM-exporter` service is disabled by default because a user might not be interested
+  in collecting metrics as this is a DCGM snap, not the DCGM-exporter snap.
+
+  **Quick Start**
+  ---
+  Getting the NVIDIA GPU metrics would look like this:
+
+  ```
+  # Install the snap
+  sudo snap install dcgm
+
+  # Start the DCGM exporter service
+  sudo snap start dcgm.dcgm-exporter
+
+  # Get the metrics
+  curl localhost:9400/metrics
+  ```
```

this should be rephrased as an exporter-specific tutorial. For example:
```suggestion
  **How to enable metrics collection**
  ---
  The DCGM Exporter is not enabled by default; if needed, it can be enabled as follows:

  \```
  # Start the DCGM Exporter service
  sudo snap start dcgm.dcgm-exporter

  # Inspect the metrics
  curl -s localhost:9400/metrics
  \```
```
...and then you can also drop the note above about the exporter being disabled by default.

Sorry for the spurious backslashes but Markdown formatting would take over otherwise